elm-package.json

```diff
@@ -1,18 +1,20 @@
 {
-  "version": "4.0.0",
-  "summary": "Run elm-test suites in a Node.js CLI. Get it with npm install -g elm-test",
-  "repository": "https://github.com/elm-community/elm-test.git",
-  "license": "BSD-3-Clause",
-  "source-directories": ["src", "native-src"],
-  "exposed-modules": [],
-  "native-modules": true,
-  "dependencies": {
-    "elm-community/elm-test": "4.0.0 <= v < 5.0.0",
-    "elm-community/json-extra": "2.0.0 <= v < 3.0.0",
-    "elm-community/lazy-list": "1.0.0 <= v < 2.0.0",
-    "elm-community/shrink": "2.0.0 <= v < 3.0.0",
-    "elm-lang/core": "5.0.0 <= v < 6.0.0",
-    "mgold/elm-random-pcg": "4.0.2 <= v < 6.0.0"
-  },
-  "elm-version": "0.18.0 <= v < 0.19.0"
+    "version": "4.0.0",
+    "summary": "Run elm-test suites in a Node.js CLI. Get it with npm install -g elm-test",
+    "repository": "https://github.com/elm-community/elm-test.git",
+    "license": "BSD-3-Clause",
+    "source-directories": [
+        "src",
+        "native-src"
+    ],
+    "exposed-modules": [],
+    "native-modules": true,
+    "dependencies": {
+        "Skinney/elm-array-exploration": "2.0.5 <= v < 3.0.0",
+        "elm-community/elm-test": "4.2.0 <= v < 5.0.0",
```

Comment:
Hm, will this break for people who are already >=0.18.3 but still have elm-community/elm-test@4.0.x or 4.1.x? Is this something we could detect and offer to fix when running this version?